src/current/v24.1/migrate-from-mysql.md

```diff
@@ -158,7 +158,7 @@ Use the [Schema Conversion Tool](https://www.cockroachlabs.com/docs/cockroachclo
 
        Click **Save**.
 
-       This is a workaround to prevent [data validation](#step-3-validate-the-migrated-data) from failing due to collation mismatches. For more details, see the [MOLT Verify] ({% link {{ page.version.version }}/molt-verify.md %}#limitations) documentation.
+       This is a workaround to prevent [data validation](#step-3-validate-the-migrated-data) from failing due to collation mismatches. For more details, see the [MOLT Verify] ({% link {{ page.version.version }}/molt-verify.md %}#known-limitations) documentation.
```

Comment:
```suggestion
       This workaround helps to prevent [data validation](#step-3-validate-the-migrated-data) from failing due to collation mismatches. For more details, refer to [MOLT Verify Known Limitations] ({% link {{ page.version.version }}/molt-verify.md %}#known-limitations).
```